### DISCUSSION TITLE
I need contributors to join my team

### DISCUSSION THREAD

**Discussion by Z-8003** [community/community#63177] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I need  team for Ibc cosmos project , can be based globally , I have investors that  will fund our operations after successful testing of our first project , send your proposals to https://github.com/ITSUMARU  , 
Regards
Kyzarzu Zikorogawa
Itsumaru (Director)

**Comment by queenofcorgis** [comment]
Hi @Z-8003, we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
